gradle/interop-as-source-set-klib.gradle

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+project.ext.registerInteropAsSourceSetOutput = { interopName, sourceSet ->
+    afterEvaluate {
+        def currentHostKotlinTarget = kotlin.targets.matching { it.platformType.name == "native" }.find {
+            it.konanTarget == hostManager.Companion.getHost()
+        }
+
+        def interop = currentHostKotlinTarget.compilations["main"].cinterops[interopName]
+        def cinteropTask = tasks.named(interop.interopProcessingTaskName)
```

Comment:
What if `currentHostKotlinTarget` was not found (equals `null`) and hence `interop` also equals `null`.
What would be the result of `tasks.named(null)` in this case?

Reply:
It's a workaround from `MPP` team and I bet that it was copy-pasted from the other projects. I will not change this file to avoid merge conflicts in future.

Reply:
Ok. Maybe at least add some report message in case of something is null or not found?